frontend/src/Editor/Inspector/Components/Chart.jsx

```diff
@@ -98,6 +97,21 @@ class Chart extends React.Component {
         currentState
       ),
     });
+    {
+      resolveReferences(this.state.component.component.definition.properties.plotFromJson?.value, currentState) &&
```

Comment:
Don't we already have this value computed in line 64. Can't we reuse the same here.?